packages/api-derive/src/types.ts

```diff
@@ -31,6 +31,15 @@ export interface DerivedContractFees {
   transferFee: BN;
 }
 
+export interface DerivedElectionsInfo {
```

Comment:
In light of this - https://github.com/polkadot-js/api/issues/1101, it probably would make more sense to . the the actual real types instead of `BN` for the new structures.